scripts/src/admin/services.ts

```diff
@@ -699,7 +699,7 @@ type UpdateAppConfigRequest = Request & {
 
 export const updateAppConfig = async function(req: UpdateAppConfigRequest, res: Response) {
 	const config = req.body;
-	const isLimitsNumbers = (obj, limitName) => typeof obj.limits[limitName] === "number";
+	const isLimitsNumbers = (obj: any, limitName: string) => typeof obj.limits[limitName] === "number";
```

Comment:
we try to avoid using `any`, only doing it when there's no other way (or the other way is too complicated), which is not the case here.
it should be `obj: LimitConfig`.

also, you shouldn't be able to bind on an arrow function, and there's also no reason to, you can just do:
```
const isLimitsNumbers = (limitName: string) => typeof config.limits[limitName] === "number";
```

as `config` is part of the closure.

Reply:
You're right about the `LimitConfig`, forgot the type already exists
About the binding: predicate with `config` is tight coupled and not functional. By patching the arguments and passing `LimitConfig` argument we can move the arrow function out of that scope

Reply:
If you move it outside then it make sense, but then you cannot use an arrow function

Reply:
Why not? I missed the point
Export this function to utils and use more than once with the patch of configObj

Reply:
it's just weird to use bind on an arrow function.  you cannot change the `this` object using bind on arrow functions, which is why it's weird.
i get that you use bind here to save the reference to the argument, but it's just weird to see it.